protocol/contracts/interfaces/v2/ITreasuryReservesVault.sol

```diff
@@ -98,24 +141,39 @@ interface ITreasuryReservesVault is ITempleElevatedAccess {
     /**
      * @notice The configuration for a given strategy
      */
-    function strategies(address strategyAddress) external view returns (
-        bool isEnabled,
+    function strategies(address strategy) external view returns (
         bool borrowPaused,
         bool repaysPaused,
         bool isShuttingDown,
-        uint256 debtCeiling,
         int256 underperformingEquityThreshold
     );
```

Comment:
This is the raw mapping getter that's added from this being public:
```
    mapping(address => StrategyConfig) public override strategies;
```

whereas the intent for `strategyDetails` is to combine more info about the underlying strategy itself, but it's more costly as it does other lookups and has a loop.

Will leave as is, just in case in future there's a write function which (efficiently) needs to lookup the mapping but without the other parts